setup.cfg

```diff
@@ -76,7 +76,7 @@ per-file-ignores =
     src/click/__init__.py: F401
 
 [mypy]
-files = src/click
+files = src/click, tests/test_shell_completion.py
```

Comment:
We're not checking tests.

Reply:
that's the only way for me to achieve this part of the contribution checklist: "Add tests that demonstrate the correct behavior of the change. Tests should fail without the change"